topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+
+As branch lengths reflect evolutionary distances, they can also be used to identify transmission clusters and outbreaks. Below is a (rooted) tree of the Central Asian Clade (CAC), which is part of lineage 2 [(Eldholm et al. 2016)](https://doi.org/10.1073/pnas.1611283113). The orange color highlights the Afghan strain family within the CAC. At the bottom of the tree, note the clade with short branch lengths. This is how one would expect an outbreak to look in a phylogenetic tree: a set of strains clustering together and separated by extremely short branches, reflecting their almost identical genomes.
+
+<img src="./images/eldholm2016_tree.png" alt="drawing" width="200"/>
+<figcaption align = "left">Fig.2 - Phylogeny of the central Asian clade, including the Oslo outbreak. Modified Eldholm et al. 2016.</figcaption>
+<br/><br/>
+
+> ### {% icon comment %} Phylogenetics with *Mycobacterium tuberculosis*
+>
+> Phylogenetics with MTB has some particularities not encountered with other organisms. Here are four points to keep in mind when doing phylogenetics with this organism.
+>
+> a) There seems to be **no horizontal gene transfer** (HGT) in the MTBC. HGT complicates phylogenetic inference in many bacteria because a piece of DNA introduced by HGT has a different history and thus phylogeny than genes not affected by HGT.
+>
+> b) There is **little genetic diversity** in the MTBC: any two strains differ by only around 2,500 SNPs over the whole genome. To achieve a good resolution of recent evolution (e.g. during an outbreak), whole genome sequences are thus extremely useful. With whole genome sequences, resolution can be further increased by not only considering SNPs, but also other types of mutations, in particular short insertions and deletions (as e.g. shown in [this study]( https://doi.org/10.1371/journal.ppat.1008357)).
+>
+> c) A large proportion of DNA polymorphisms in the MTBC are **singletons**, that is, variants present only in a single strain. This adds to the problem of low diversity, since singletons are not informative about tree topology.
+>
+> d) In this workshop, and indeed in many studies of the MTBC, SNPs are called not against H37Rv, but against a reconstructed ancestral genome. This means that the number of SNPs identified does not reflect the evolutionary distance from some random strain like H37Rv, but from the most recent common ancestor of the MTBC. Below we will see that this has implications for the interpretation of a tree.
```

Comment:
I always rather like Figure 3 from [this preprint](https://www.biorxiv.org/content/10.1101/403824v1.full) for this point.

Reply:
Indeed! I added a reference to the figure in the preprint